plugins/experimental/txn_box/plugin/src/query.cc

```diff
@@ -88,12 +88,9 @@ query_take_qpair(TextView &qs)
   // If there's anything left, look for kv pair.
   if (qs) {
     auto v{qs.clip_prefix_of([](char c) { return c != '&' && c != ';'; })};
-    auto k = v.take_prefix_at('=');
+    auto k = v.take_prefix_at(QPair::Sep);
     QPair zret{k, v};
     zret.elt_sep = elt_sep;
-    if (v.begin() > k.end()) {
-      zret.kv_sep = v[-1];
```

Comment:
I think this can be ignored, it is using the same separator to find(`take_prefix_at`) and to place it here, so probably is the same just to leave the default which will not be overwritten.